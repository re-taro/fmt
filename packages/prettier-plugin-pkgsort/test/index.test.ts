import { format } from "prettier";
import { describe, expect, it } from "vitest";

// TODO

describe("should", () => {
	it("sort", async () => {
		await expect(
			format(
				`
{
  "name": "foo",
  "version": "1.0.0",

  "dependencies": {

   }
  }


  `,
				{
					plugins: ["./dist/index.js"],
					filepath: "foo/package.json",
				},
			),
		).resolves.toMatchSnapshot();
	});
});
