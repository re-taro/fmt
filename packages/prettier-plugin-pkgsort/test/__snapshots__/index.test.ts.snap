// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`should > sort 1`] = `
"{
  \\"name\\": \\"foo\\",
  \\"version\\": \\"1.0.0\\",
  \\"dependencies\\": {}
}
"
`;
